tests/model/test_model.py

```diff
@@ -656,71 +656,110 @@ def test_append_message(self, mocker, user_profile, response,
         # LOG REMAINS THE SAME IF UPDATE IS FALSE
         assert model.msg_list.log == log
 
-    @pytest.mark.parametrize('response, update_call_count, expected_message', [
+    @pytest.mark.parametrize('response, update_call_count, new_index', [
         ({
             'subject': 'new subject',
             'message_ids': [1],
         }, 1, {
-            'content': 'old content',
+            'messages': {
```

Comment:
For the changes in this third commit, are you broadening the test to show how surrounding messages are affected?

Reply:
yes